README.md

```diff
@@ -12,48 +12,64 @@ Python with the scientific stack, compiled to WebAssembly.
 
 ## What is Pyodide?
 
-**Pyodide** brings the Python 3.8 runtime to the browser via WebAssembly, along with the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and scikit-learn. The [`packages` directory](https://github.com/pyodide/pyodide/tree/master/packages) lists over 75 packages which are currently available. In addition it's possible to install pure Python wheels from PyPi.
+Pyodide may be used in any context where you want to run Python inside a web
+browser.
 
-**Pyodide** provides transparent conversion of objects between Javascript and Python.
-When used inside a browser, Python has full access to the Web APIs.
+Pyodide brings the Python 3.8 runtime to the browser via WebAssembly, along with
+the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and
+scikit-learn. The [packages directory](packages) lists over 75 packages which
```

Comment:
At some point it'd be good to have a packages index page as suggested in #1380 and then we could link that here.